docs/fsharp/language-reference/reference-cells.md

```diff
@@ -101,6 +98,45 @@ The following code examples demonstrate the use of reference cells in closures.
 
 [!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-1/snippet2207.fs)]
 
+## Consuming C#7 and VB15 `ref` returns
+
+Starting with F# 4.1, you can consume `ref` returns generated in C#7 or VB15.  The result of such a call is a reference cell.
```

Comment:
Change to "The result of such a call is a ``byref<_>`` pointer."  

* A reference cell is a ``ref<_>`` = ``FSharpRef<_>`` heap-allocated object with a single mutable field (which you can take the address of as a byref). 
* A byref is a safe managed pointer.